raster/r.colors.out/raster3d_main.c

```diff
@@ -55,6 +60,28 @@ int main(int argc, char **argv)
     opt.file->description = _("If not given write to standard output");
     opt.file->required = NO;
 
+    opt.format = G_define_standard_option(G_OPT_F_FORMAT);
+    opt.format->guisection = _("Print");
+
+    opt.color_format = G_define_option();
```

Comment:
How about moving this to the parser as a new `G_define_standard_option()` macro?

Reply:
done

Reply:
A new parser option should be an issue for a separate PR, shouldn't it? (Considering our squash-PR policy).

Reply:
@neteler, if that's the case, should I roll back the current parser option and create a new issue once the PR is merged?

Reply:
Sure, that's fine.

Reply:
I am not sure a new parser macro was needed here in the first place, no other tool can use it since the color conversion is implemented only here, not in the lib, but I suggest just move on.